src/driver/mongodb/MongoConnectionOptions.ts

```diff
@@ -245,4 +245,8 @@ export interface MongoConnectionOptions extends BaseConnectionOptions {
      */
     readonly loggerLevel?: "error"|"warn"|"info"|"debug";
 
+    /**
+     * Specifies the list of authentication mechanisms the server accepts.
```

Comment:
Description is little bit strange. What about "**Sets the authentication mechanism that MongoDB will use to authenticate the connection.**". Also there is `docs/connection-options.md` file with all options.

Reply:
Oh, I forgot to change it after copy/paste. Sure, I fix it!

Reply:
Nice :)